packages/ember-template-compiler/tests/plugins/assert-input-helper-without-block-test.js

```diff
@@ -1,16 +1,17 @@
 import { compile } from '../../index';
+import { moduleFor, AbstractTestCase } from 'internal-test-helpers';
 
-QUnit.module('ember-template-compiler: assert-input-helper-without-block');
+moduleFor('ember-template-compiler: assert-input-helper-without-block', class extends AbstractTestCase {
+  ['@test Using {{#input}}{{/input}} is not valid'](assert) {
+    assert.expect(1);
```

Comment:
Most of the time we can get rid of `assert.expect(1)` and I believe we can in these test cases. We will want to keep the `assert.expect(0)` however